demo/demo-springmvc/springmvc-client/src/main/java/org/apache/servicecomb/demo/springmvc/client/SpringmvcClient.java

```diff
@@ -87,10 +87,10 @@ public static void run() {
       Map<String, Double> metrics = restTemplate.getForObject(prefix + "/metrics", Map.class);
 
       TestMgr
-          .check(true, metrics.get("jvm(statistic=gauge,name=heapUsed)") != 0);
+          .check(true, metrics.get("jvm(name=heapUsed,statistic=gauge)") != 0);
       TestMgr.check(true, metrics.size() > 0);
       TestMgr.check(true, metrics.get(
-          "servicecomb.invocation(operation=springmvc.codeFirst.saySomething,role=producer,stage=whole,statistic=count,status=200)")
+          "servicecomb.invocation(operation=springmvc.codeFirst.saySomething,role=PRODUCER,stage=total,statistic=count,status=200)")
```

Comment:
the PRODUCER is upper case, but other tags are lower case.

Reply:
Done,add toLowerCase()